gradle-plugin/it/src/it/openliberty/gradle.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+http.port=9080
+https.port=9443
```

Comment:
Could you do a proper setting for these values just like we do in the quickstart. The same ports are hardcoded at several places (build.gradle, gradle.properties, server.xml) and it should only be present in this file.

Reply:
https://github.com/eclipse/jkube/pull/1541/files#r916649884